docs/ens.md

```diff
@@ -0,0 +1,104 @@
+---
+id: ens
+title: Adding accounts to an ENS domain
+sidebar_label: Using ENS with DOT/KSM accounts
+---
+
+ENS (Ethereum Name Service) is a system of smart contracts on the Ethereum blockchain which allows
+users to claim domain names like `bruno.eth`. Supporting wallets can then allow senders to input ENS
+domains instead of long and unwieldy addresses. This prevents phishing, fraud, typos, and adds a
+layer of usability on top of the regular wallet user experience.
+
+> Note: You will need an ENS name and an Ethereum account with some ether in it to follow along with
+> this guide. To register an ENS name, visit the [ENS App](https://app.ens.domains) or any number of
+> subdomain registrars like [Nameth](https://nameth.io). You will also need some way to use the
+> Ethereum address - we recommend [Frame](https://frame.sh/) or [Metamask](https://metamask.io).
```

Comment:
Why do you recommend these two wallets for Ethereum?

Reply:
Because they make it very easy to interact with any web3 enabled website, and are basically the only option on desktops (where we want users to do this process, since it's very clumsy on mobile)

Reply:
Ah i see -- should you add this in? I wonder if other users might also be thinking "why is Web3 recommending these wallets to me?"

Reply:
Yeah, could be clearer, thanks, fixing